src/locales/en.json

```diff
@@ -4,67 +4,67 @@
     "explanationcard": {
       "person": "person",
       "person_plural": "people",
       "multiple_suffix": "x your weekly risk budget",
       "percentage_suffix": "% of your weekly risk budget",
-      "details_header_closed": "Show calculation details",
-      "details_header_open": "Hide calculation details",
+      "details_header_closed": "Show details",
+      "details_header_open": "Hide details",
       "details_overview_header": "Calculation",
       "details_person_risk": "Person Risk",
       "details_activity_risk": "Activity Risk",
       "details_steps_header": "Calculation Steps",
       "details_person_risk_number": "{{ person_risk }}-in-a-million chance",
-      "details_person_risk_explanation": "First, we calculate that each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID.",
-      "details_person_risk_explanation_housemate": "First, we calculate that each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID, not including their risk from living with you.",
+      "details_person_risk_explanation": "First, the calculation assumes each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID.",
+      "details_person_risk_explanation_housemate": "First, the calculation assumes each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID, not including their risk from living with you.",
       "chance": "chance",
-      "details_activity_risk_explanation": "Next, we calculate the risk of the activity. A person at this activity with COVID would have a <strong>{{ activity_risk }} chance</strong> of transmitting it to you.",
+      "details_activity_risk_explanation": "Next, the activity risk is calculated from an estimation of a person with COVID-19 having a <strong>{{ activity_risk }} chance</strong> of transmitting it to you.",
       "details_number_of_people": "Number of people",
       "details_total_risk": "Total risk",
       "details_total_risk_number": "{{ points }}-in-a-million ({{ percentage }})",
-      "details_total_risk_explanation": "Finally, we multiply Person Risk, Activity Risk, and the number of people to get the total result of roughly <strong>{{ points }}-in-a-million ({{ percentage }})</strong> chance of getting COVID from this activity with these people <strong>{{ frequency }}</strong>.",
+      "details_total_risk_explanation": "Finally, the personal and activity risk, and the number of people to get the total result of roughly <strong>{{ points }}-in-a-million ({{ percentage }})</strong> chance of getting COVID-19 from this activity with these people <strong>{{ frequency }}</strong> is multiplied.",
```

Comment:
Our style is to usually just say "COVID", not "COVID-19".

Reply:
There is a bonus here of being correct. Besides, there might be a COVID-20 or -21 later on.

Reply:
Ah, I think an important thing you're missing is that the words we use on the calculator in English are following style trends we also use in the white paper in English. So there are more words than just the ones you see here, and we use them in specific ways.

For example I also notice you changed "Person Risk, Activity Risk" to "the personal and activity risk". However, Person Risk has a highly specific meaning that is explained in the white paper, and it is capitalized because it is a technical term. So we cannot change this.

Reply:
Aha, shouldn't that be "Personal Risk" still? Can the white paper be changed?